confd/templates/janitor.properties.tmpl

```diff
@@ -1,2 +1,191 @@
+# Janitor Monkey runs every day at 11am.
+simianarmy.calendar.openHour = {{getv "/simianarmy/calendar/openhour" "11"}}
+simianarmy.calendar.closeHour = {{getv "/simianarmy/calendar/closehour" "11"}}
+simianarmy.calendar.timezone = {{getv "/simianarmy/calendar/timezone" "America/Los_Angeles"}}
+
+# Let Janitor Monkey run
+simianarmy.janitor.enabled = {{getv "/simianarmy/janitor/enabled" "true"}}
+
+# Don't allow Janitor Monkey to change resources (dryrun mode)
+simianarmy.janitor.leashed = {{getv "/simianarmy/janitor/leashed" "true"}}
+
+# The SDB domain for storing the resources managed by the Janitor Monkey.
+simianarmy.janitor.resources.sdb.domain = {{getv "/simianarmy/janitor/resources/sdb/domain" "SIMIAN_ARMY"}}
+
+# override to force monkey time, useful for debugging off hours
+{{if exists "/simianarmy/calendar/ismonkeytime" -}}
+simianarmy.calendar.isMonkeyTime = {{getv "/simianarmy/calendar/ismonkeytime" "true"}}
+{{end -}}
+#simianarmy.calendar.isMonkeyTime = true
+
+# Currently Janitor Monkey can clean up the following resources
+simianarmy.janitor.enabledResources = {{getv "/simianarmy/janitor/enabledresources" "Instance, ASG, EBS_Volume, EBS_Snapshot, Launch_Config"}}
+
+# The property below needs to be a valid email address to send notifications for Janitor Monkey
+simianarmy.janitor.notification.sourceEmail = {{getv "/simianarmy/janitor/notification/sourceemail" "foo@bar.com"}}
+
+# The property below needs to be a valid email address to receive the summary email of Janitor Monkey
+# after each run
+simianarmy.janitor.summaryEmail.to = {{getv "/simianarmy/janitor/summaryemail/to" "foo@bar.com"}}
+
+# The property below needs to be a valid email address to receive the notifications of Janitor Monkey
+# for resouces that do not have a valid owner email specified
+simianarmy.janitor.notification.defaultEmail = {{getv "/simianarmy/janitor/notification/defaultemail" "foo@bar.com"}}
+
+# The property below specifies the number of business days that a notification is sent before the
+# expected termination time. For example, if a resource is scheduled to be cleaned up by Janitor
+# Monkey on 12/13/2012, Thursday and the property is set to 2, the owner will receive notification
+# about the cleanup on 12/11/2012, Tuesday, which is 2 business days before the termination date.
+simianarmy.janitor.notification.daysBeforeTermination = {{getv "/simianarmy/janitor/notification/daysbeforetermination" "2"}}
+
+# The owner id that snapshots have for being managed by Janitor Monkey. This property needs
+# to set if you use Edda for getting snapshots to avoid getting shared snapshots. If you are using
+# AWS to get the snapshots, this property does not need to be set.
+{{if exists "/simianarmy/janitor/snapshots/ownerid" -}}
+simianarmy.janitor.snapshots.ownerId = {{getv "/simianarmy/janitor/snapshots/ownerid" "012345678"}}
+{{end -}}
+#simianarmy.janitor.snapshots.ownerId = 012345678
```

Comment:
Yeah some of these comments were mainly so I could check that my script was working as expected - I'll remove